### DISCUSSION TITLE
Verify stalled after google auth login completes

### DISCUSSION THREAD

**Discussion by amberfi** [Web3Auth/Web3Auth#1291] [Help]
For some reason the verify stalls consistently on our site.  Any idea what could cause this?  If we refresh the page, the connection continues to happen, but it almost always stalls.

<img width="407" alt="Screenshot 2023-02-06 at 4 02 14 PM" src="https://user-images.githubusercontent.com/98486422/217114984-5314f152-1349-418a-83f8-568e7d0dddbe.png">

**Comment by YZhenY** [comment]
relevant console logs 
<img width="1326" alt="Screenshot 2023-02-06 at 4 10 41 PM" src="https://user-images.githubusercontent.com/25172440/217135565-8df032bf-e2b2-4952-b99c-9be1f222f993.png">

**Comment by tanguyenvn** [comment]
We could only reproduce the error once and then it didn't happen anymore which means the issue is random. 
I suppose you're enabling Torus Wallet plugin when integrating Web3Auth SDK. From the console logs, we can see that loading Torus wallet failed and caused the Login modal to stall. There can be several reasons that cause Torus Wallet failure e.g. race condition
Please updgrade @web3auth/modal package to the version 4.4.2. The login modal should not stall even when Torus wallet plugin fails to load.
